go.mod

```diff
@@ -207,6 +208,8 @@ require (
 
 replace google.golang.org/grpc v1.54.0 => google.golang.org/grpc v1.26.0
 
+replace github.com/tikv/pd/client => ./client
```

Comment:
Will it have a cycle dependency problem?

Reply:
Do you think it's a better way to support a seperate gomod for pd-ctl?

Reply:
I think it seems feasible, but I'm not sure if it's a best practice

Reply:
I am not sure whether have better way as well...

Reply:
How about using a separate go mod for pd-ctl like other tools?

Reply:
supported by https://github.com/tikv/pd/pull/7635